sdk/samples/iot/templates/iothub/test-resources.json

```diff
@@ -0,0 +1,96 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "The base resource name."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "westus2"
+        },
+        "sku_name": {
+            "type": "string",
+            "defaultValue": "S1"
+        },
+        "sku_units": {
+            "type": "string",
+            "defaultValue": "1"
+        },
+        "d2c_partitions": {
+            "type": "string",
+            "defaultValue": "4"
+        },
+        "features": {
+            "type": "string",
+            "defaultValue": "none"
+        },
+        "tags": {
+            "type": "object",
+            "defaultValue": {}
+        },
+        "cloudEnvironment": {
+            "type": "string",
+            "metadata": {
+                "description": "Cloud environment to deploy (i.e. usgov/china/ ...)"
+            },
+            "allowedValues": [
+                "public",
+                "china",
+                "usgov"
+            ],
+            "defaultValue": "public"
+        }
```

Comment:
This isn't referenced anywhere in the ARM template. I think it can be removed without issue.